lua/telescope/builtin/internal.lua

```diff
@@ -350,7 +350,9 @@ internal.commands = function(opts)
 end
 
 internal.quickfix = function(opts)
-  local locations = vim.fn.getqflist()
+  opts = opts or {}
```

Comment:
you dont need this for builtins. We wrap builtins at the end of each module, so we never even have a chance to not include it